src/extension.ts

```diff
@@ -78,9 +50,9 @@ export function activate(context: vscode.ExtensionContext) {
         return;
       }
 
-      const selectedComponent = selectedComponents.map((component) => component.label);
+      const selectedComponentsLabel = selectedComponents.map((component) => component.label);
 
-      const installCmd = await getInstallMultipleComponents(selectedComponent.join(' '));
```

Comment:
@selemondev I think we can have a separate command for multiple installs as `shadcn-ui.addMultipleComponents`
I would suggest you revert the last change and just update `getInstallMultipleComponents(selectedComponent.join(' '))` with `getInstallCmd(selectedComponentsLabel)`

Reply:
@SuhelMakkad, the first pull request I created adds the capability to install multiple components simultaneously using the VSCode Quick Pick window with checkboxes. Here's my reasoning:

1. Since we already have the normal way for installing components via the VSCode window (shadcn/ui: add a new component - for a single component), I believed it would be more efficient to introduce a separate command for installing multiple components simultaneously.

2. In the first pull request, I integrated this feature of adding multiple components simultaneously. I implemented a function, maintaining separation of concerns. Although the function resembles the one for installing a single component, the key difference lies in the additional option passed to the VSCode Window API: the `canPickMany` property. This property allows developers to select components by checking checkboxes.

3. The subsequent refactored pull request eliminates the `getInstallMultipleComponents` command, integrating it into the `getInstallCmd`. Now, when a developer selects the `shadcn/ui: add new component` command, a window displays with checkboxes beside each component label. This allows developers to choose and install the desired components.

4. Instead of having two separate commands, namely `shadcn/ui: add new component` and `shadcn/ui: add multiple components`, we now have a single command that accommodates both functionalities 😁.

Reply:
I somewhat agree. When I added the `shadcn/ui: add new component` command I started with multi picker only.

After using the extension for some time I realized that if I wanted to install just one component, then I had to search for the component and then click the checkbox and then click on the run command. That is three steps to install one command. Later I just dropped the multipicker. Because most of the time when a user is using shadcn/ui they are just installing one component at a time (mostly).

What I am suggesting is we add two different commands `shadcn/ui: add new component` and `shadcn/ui: add multiple components` which is what you had earlier. 

The only feedback I had was that inside the `shadcn-ui.addMultipleComponents` disposable, you had created a function as `getInstallMultipleComponents` which can be replaced by `getInstallCmd` as it already accepts an array of components.

So this line
```ts
const installCmd = await getInstallMultipleComponents(selectedComponent.join(' '));
```
can be replaced with 
```ts
const installCmd = await getInstallCmd(selectedComponent);
```

Let me know what you think.